langchain-core/src/prompts/chat.ts

```diff
@@ -793,8 +825,17 @@ export class ChatPromptTemplate<
 
   validateTemplate = true;
 
+  templateFormat: TemplateFormat = "f-string";
+
   constructor(input: ChatPromptTemplateInput<RunInput, PartialVariableName>) {
     super(input);
+    // If input is mustache and validateTemplate is not defined, set it to false
```

Comment:
Why?

Reply:
Since `this.validateTemplate` defaults to `true`, if `undefined` is passed it'll be assigned `true`.